app/admin.js

```diff
@@ -8,10 +8,8 @@ const pjson = require('../package.json');
 const fs = require('fs');
 const crypto = require('crypto');
 
-const db = require('./db.js');
-const MasterKey = require('./models/masterkey.js');
```

Comment:
instead of importing the DB, MasterKey file, and WalletKey file, we only import these if necessary. The reason is that these files require a DB connection, which is impossible and not allowed in an offline environment. Importing these files has been moved to a helper function below called "requireDB"